scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute the statistics (mean, stddev) of scalar maps, which can represent
+diffusion metrics, in a ROI.
+
+The mask can either be a binary mask, or a weighting mask. If a weighting mask
+should either contain floats between 0 and 1, or should be normalized with
+--normalize_weights.
+
+IMPORTANT: if the mask contains weights (and not 0 and 1 exclusively), the
+standard deviation will also be weighted.
+"""
+
+import argparse
+import nibabel as nib
+import numpy as np
+import os
+import json
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_json_args,
+                             assert_inputs_exist)
+from scilpy.utils.filenames import split_name_with_nii
+from scilpy.utils.metrics_tools import get_roi_metrics_stats
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_mask',
+                   help='Mask volume file name, formatted in any nibabel ' +
+                        'supported format.\nCan be a binary mask or a ' +
+                        'weighted mask.')
+
+    g = p.add_mutually_exclusive_group(required=True)
+    g.add_argument('--metrics_dir',
+                   help='Metrics files directory. name of the directory ' +
+                        'containing the metrics files.')
+    g.add_argument('--metrics', dest='metrics_file_list', nargs='+',
+                   help='Metrics nifti file name. list of the names of the ' +
+                        'metrics file, in nifti format.')
+
+    p.add_argument('--bin', action='store_true',
+                   help='If set, will consider every value of the mask ' +
+                        'higher than 0 to be part of the mask, and set to 1 ' +
+                        '(equivalent weighting for every voxel).')
+
+    p.add_argument('--normalize_weights', action='store_true',
+                   help='If set, the weights will be normalized to the [0,1] '
+                        'range.')
+
```

Comment:
bin and normalize_weights should probably be exclusive. Or test lower that user did not use both.